tools/accuracy_checker/accuracy_checker/metrics/reid.py

```diff
@@ -330,6 +330,55 @@ def evaluate(self, annotations, predictions):
 
         return [(tp+tn) / (tp+fp+tn+fn)]
 
+class NormalizedEmbeddingAccuracy(FullDatasetEvaluationMetric):
+    """
+    Accuracy score calculated with normalized embedding dot products
+    """
+    __provider__ = 'normalized_embedding_accuracy'
+
+    annotation_types = (ReIdentificationClassificationAnnotation, ReIdentificationAnnotation, )
+    prediction_types = (ReIdentificationPrediction, )
+
+    def submit_all(self, annotations, predictions):
+        return self.evaluate(annotations, predictions)
+
+    def evaluate(self, annotations, predictions):
+        true_positive = false_positive = 0
+        gallery_embeddings = extract_embeddings(annotations, predictions, query=False)
+        query_start_idx = len(gallery_embeddings)
+
+        for ann, pred in zip(annotations[query_start_idx:], predictions[query_start_idx:]):
+            best_sim = 0
+            pred_person_id = -1
+
+            person_id = ann.person_id
+            camera_id = ann.camera_id
```

Comment:
you specify `ReidentificationClassificationAnnotattion` as suitable data type, but it does not have such fields, please remove it from `annotation_types`